lxd/api_cluster.go

```diff
@@ -4599,3 +4618,675 @@ func autoHealCluster(_ context.Context, s *state.State, offlineMembers []db.Node
 
 	return nil
 }
+
+// swagger:operation GET /1.0/clusters cluster-links cluster_links_get
+//
+//	Get cluster links
+//
+//	Gets cluster links.
+//
+//	---
+//	produces:
+//	  - application/json
+//	responses:
+//	  "200":
+//	    description: Cluster links
+//	    schema:
+//	      type: object
+//	      description: Sync response
+//	      properties:
+//	        type:
+//	          type: string
+//	          description: Response type
+//	          example: sync
+//	        status:
+//	          type: string
+//	          description: Status description
+//	          example: Success
+//	        status_code:
+//	          type: integer
+//	          description: Status code
+//	          example: 200
+//	        metadata:
+//	          $ref: "#/definitions/ClusterLink"
+//	  "403":
+//	    $ref: "#/responses/Forbidden"
+//	  "500":
+//	    $ref: "#/responses/InternalServerError"
+func clusterLinksGet(d *Daemon, r *http.Request) response.Response {
```

Comment:
Recursion query parameter:
1. No recursion or `recursion=0`:
```json
[
   "/1.0/clusters/cluster1",
   "/1.0/clusters/cluster2"
]
```
2. Recursion > 0 = what you have now :)